client/state/sharing/publicize/actions.js

```diff
@@ -71,6 +75,44 @@ export function fetchConnections( siteId ) {
 }
 
 /**
+ * Triggers a network request to request a Publicize connection for the
+ * specified site ID.
+ *
+ * @param  {Number} siteId       Site ID
+ * @param  {Number} connectionId ID of the connection to be fetched.
+ * @param  {String} label        Name of the service to be fetched.
+ * @return {Function}            Action thunk
+ */
+export function fetchConnection( siteId, { ID: connectionId, label } ) {
```

Comment:
It's not clear from the changes here the intended usage of the second argument. How would we know the label to pass if we've yet to fetch the connection? Ideally this would take simple arguments like `siteId, connectionId` .